pds4indextools/pds4_create_xml_index.py

```diff
@@ -146,70 +189,22 @@ def load_config_file(specified_config_file):
-
-    Inputs:
-        xsd file                  An XML Schema Definition file.
-        nillable_elements_info    A dictionary containing nillable element information.
-    """
-    tree = etree.fromstring(requests.get(xsd_file).content)
-    namespace = {'xs': 'http://www.w3.org/2001/XMLSchema'}
-
-    elements_with_nillable = tree.xpath('//xs:element[@nillable="true"]',
-                                        namespaces=namespace)
-
-    for element in elements_with_nillable:
-        name = element.get('name')
-        type_attribute = element.get('type')
-        if type_attribute not in nillable_elements_info.keys():
-            if type_attribute:
-                # Split the type attribute to handle namespace:typename format
-                type_parts = type_attribute.split(':')
-                # Take the last part as the type name
-                type_name = type_parts[-1]
-
-                # Attempt to find the type definition in the document
-                type_definition_xpath = (f'//xs:simpleType[@name="{type_name}"] | '
-                                         f'//xs:complexType[@name="{type_name}"]')
-                type_definition = tree.xpath(
-                    type_definition_xpath, namespaces=namespace)
-
-                if type_definition:
-                    # Take the first match
-                    type_definition = type_definition[0]
-                    base_type = None
-                    # For complexType with simpleContent or simpleType, find base attr
-                    if type_definition.tag.endswith('simpleType'):
-                        restriction = type_definition.find('.//xs:restriction',
-                                                           namespaces=namespace)
-                        if restriction is not None:
-                            base_type = restriction.get('base')
-                    elif type_definition.tag.endswith('complexType'):
-                        extension = type_definition.find('.//xs:extension',
-                                                         namespaces=namespace)
-                        if extension is not None:
-                            base_type = extension.get('base')
-
-                    nillable_elements_info[name] = (
-                        base_type or 'External or built-in type')
-                else:
-                    # Type definition not found, might be external or built-in type
-                    nillable_elements_info[name] = 'External or built-in type'
-
 
 def process_schema_location(file_path):
     """Process schema location from an XML file.
 
-    Args:
+    Inputs:
         file_path    Path to the XML file.
 
     Returns:
         List of XSD URLs extracted from the schema location.
     """
     # Load and parse the XML file
-    tree = etree.parse(file_path)
-    root = tree.getroot()
+    try:
+        tree = etree.parse(file_path)
+        root = tree.getroot()
+    except OSError:
```

Comment:
Do both of these statements potentially trigger an `OSError`? If not, the other should be outside the try/except block.